src/backend/distributed/citus--6.1-12--6.1-13.sql

```diff
@@ -29,7 +24,8 @@ CREATE TYPE colocation_placement_type AS (
 -- tables_colocated returns true if given tables are co-located, false otherwise.
 -- The function checks shard definitions, matches shard placements for given tables.
 --
-CREATE OR REPLACE FUNCTION citus_tables_colocated(table1 regclass, table2 regclass)
+CREATE OR REPLACE FUNCTION pg_catalog.citus_tables_colocated(table1 regclass,
```

Comment:
Why did this function retain the `citus_` prefix? Is `tables_colocated` too generic (I'm actually fine with it, or maybe `tables_are_colocoated` or `distributed_tables_colocated`?

It appears the comment says `tables_colocated` even though the method retains the `citus_` prefix.